templates/wf_footer.html

```diff
@@ -57,5 +57,26 @@
     analytics.page();
   }}();
 </script>
+{% block end_body %}
```

Comment:
please remove this script as well. as it would be included in all docs pages, not only the download page. just skip tracking for now.